drivers/png/image_loader_png.cpp

```diff
@@ -29,368 +29,83 @@
 /*************************************************************************/
 
 #include "image_loader_png.h"
+#include "png_driver_common.h"
```

Comment:
This should be in the next block with the core includes as per http://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#header-includes.